### DISCUSSION TITLE
Julia REPL

### DISCUSSION THREAD

**Discussion by YaelMtz** [zed-industries/zed#20522] [Feature Requests]
Hi, I'm a physics undergrad student and I'm new to the coding world but I like to explore new technologies. I do mainly scientific computing. The REPL using the Jupyter Kernel for The Julia Language is not loading. It says it is queued but it does nothing. Is the REPL feature still not fully working?

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
